addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2175,7 +2208,13 @@ options.registry.collapse = options.Class.extend({
 options.registry.WebsiteLevelColor = options.Class.extend({
     specialCheckAndReloadMethodsNames: options.Class.prototype.specialCheckAndReloadMethodsNames
         .concat(['customizeWebsiteLayer2Color']),
-
+    /**
+     * @constructor
+     */
+    init() {
+        this._super(...arguments);
+        this._rpc = options.serviceCached(this.bindService("rpc"));
```

Comment:
Not entirely sure all RPC of this class should be cached... but I'll trust you for now :+1: At worse, this can be easily fixed and should not cause too much of an issue.